### DISCUSSION TITLE
Unmanaged package of Coe

### DISCUSSION THREAD

**Discussion by Aravindganesan** [microsoft/powerapps-tools#1002] [Q&A] [answered]
Hi, 
I am setting up the CoE in one of our lower environments. The CoE does a great job in reporting. However, there are specific needs that would be helpful to configure, Given that CoE comes as a managed package, it makes it hard to customize the changes.

Keeping in mind that there will be further updates that we may have to install. What is the best way to move forward with customization?
Is there an unmanaged package of CoE that is available

Appreciate your help

**Comment by manuelap-msft** [comment, marked as answer]
Here are our instructions for extending the kit: https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup#extending-the-starter-kit - it's through adding an unmanaged layer on your side.

**Reply by Aravindganesan** [reply]
Thanks! Given that Flows and Canvas apps cannot be edited. 

The suggestion is to create a copy and edit in an unmanaged solution.
Since its a copy I am assuming you cannot apply on the same app/flow. Is the best practice to disable the customized flows in the CoE starter kit and use the customized oness?

Also, how does it affect future upgrades?
